traffic_ops/traffic_ops_golang/trafficstats/stats_summary.go

```diff
@@ -154,8 +154,8 @@ func CreateStatsSummary(w http.ResponseWriter, r *http.Request) {
 	}
 
 	successMsg := "Stats Summary was successfully created"
-	logMsg := fmt.Sprintf("STATSSUMMARY: %v, ID: %v, ACTION: create stats_summary", *ss.StatName, id)
-	api.CreateChangeLogRawTx(api.ApiChange, logMsg, inf.User, inf.Tx.Tx)
+	// logMsg := fmt.Sprintf("STATSSUMMARY: %v, ID: %v, ACTION: create stats_summary", *ss.StatName, id)
```

Comment:
please delete. no need to leave cruft in the code. if this code is needed, easy enough to revert it using git.

Reply:
Thanks. I was just testing and looks like I left this in.